src/platform/term.c

```diff
@@ -887,6 +890,32 @@ int term_keycodeByName(const char *name) {
     return name[0];
 }
 
+static int term_ctrlPressed(int* key) {
+    if (*key == 13) {
+        *key = RETURN_KEY;
+        return 0;
+    }
+    if (*key == '\t') { // Tab is represented as "^I"
+        return 0;
+    }
+    const char* str = keyname(*key);
+    if (str == 0) {
```

Comment:
```suggestion
    if (str == NULL) {
```
? or is it documented to give 0?

Reply:
Done.